internal/fullnode/labels_test.go

```diff
@@ -0,0 +1,30 @@
+package fullnode
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	cosmosv1 "github.com/strangelove-ventures/cosmos-operator/api/v1"
+)
+
+func TestDefaultLabels(t *testing.T) {
+	t.Parallel()
+
+	t.Run("give full node", func(t *testing.T) {
+		var crd cosmosv1.CosmosFullNode
+		crd.Spec.Type = "FullNode"
+
+		labels := defaultLabels(&crd)
```

Comment:
Actually, this is a private method, of which we shouldn't test. In this codebase, we only test the public interfaces/apis. Other unit tests should have failed (expected) to hint where to add and test this label.